modules/ravestate_roboyqa/__init__.py

```diff
@@ -21,9 +21,9 @@
 
 with Module(name="roboyqa", config={ROBOY_NODE_CONF_KEY: 356}):
 
-    @state(cond=s("idle:bored"), write="rawio:out", weight=1.1, cooldown=30.)
-    def hello_world_roboyqa(ctx):
-        ctx["rawio:out"] = "Ask me something about myself!"
+    # @state(cond=s("idle:bored"), write="rawio:out", weight=.0, cooldown=30.)
+    # def hello_world_roboyqa(ctx):
+    #    ctx["rawio:out"] = "Ask me something about myself!"
 
```

Comment:
This should be uncommented again, as active engagement for roboyqa should be there

Reply:
👍